frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1034,21 +1034,33 @@ const ManageHostsPage = ({
     setIsUpdatingHosts(true);
 
     const teamId = typeof transferTeam.id === "number" ? transferTeam.id : null;
-    const currentTeam = teamIdForApi;
 
-    let action = hostsAPI.transferToTeam(teamId, selectedHostIds);
-
-    if (isAllMatchingHostsSelected) {
-      const labelId = selectedLabel?.id;
-
-      action = hostsAPI.transferToTeamByFilter({
-        teamId,
-        query: searchQuery,
-        status,
-        labelId,
-        currentTeam,
-      });
-    }
+    const action = isAllMatchingHostsSelected
```

Comment:
Fixes bug where transfer and transfer/filter endpoint are both being hit (#12247 Fix was made to delete host endpoint, but since transfer didn't cause a bug, it didn't get looked at)